broker-daemon/orderbook/index.js

```diff
@@ -217,6 +217,28 @@ class Orderbook {
     })
   }
 
+  async getAveragePrice (orders, targetDepth) {
```

Comment:
`getAveragePrice` should probably just retrieve the orders directly unless there is a good reason for it not to